conanfile_ess_mpi.txt

```diff
@@ -1,21 +1,44 @@
 [requires]
-cmake_findboost_modular/1.69.0@bincrafters/stable
-boost_filesystem/1.69.0@bincrafters/stable
-hdf5/1.10.5-dm2@ess-dmsc/stable
-gtest/1.10.0
+boost/1.69.0
+hdf5/1.10.6
```

Comment:
It looks like centos7 does not work with hdf5/1.10.6 (for unknown reason). As a workaround you can downgrade it into `hdf5/1.10.5-dm2@ess-dmsc/stable` back and create a new ticket for it.

Reply:
Ah. Thanks. I will have a look on this. I was already asking if we can drop Centos7 in particular since Centos no longer exists so we have to put quite some effort in supporting an even no longer existing distribution. For some reason this seems to be still important for ESS. But maybe we should discuss this with the ESS folks again.